### DISCUSSION TITLE
Share ideas for new features

### DISCUSSION THREAD

**Discussion by yolanmees** [yolanmees/Spikster#5] [Ideas]
I'm going to start with adding new features.

If anyone would like to see a feature added, feel free to drop it here!

**Comment by donniewr** [comment]
I believe you should have simply updated CIPI, Ubuntu 22.04 support, PHP 8.2, and 8.3, etc. The last time I used Spikster, it was very buggy.

**Reply by yolanmees** [reply]
Hi @donniewr 

My initial intention was to simply update CIPI, however upon delving deeper it became apparent that this was just the tip of the iceberg. It was clear that CIPI required a more comprehensive overhaul to realize its full potential.

Spikster's reliance on SSH for performing routine tasks for example. Using a server's own SSH protocol to self-modify, is not reflective of the robust and efficient behavior I aim to. To address this, I am transitioning Spikster to a more secure and scalable approach. Specifically, I am developing executables that are accessible via a well-defined API, thereby enhancing Spikster's operational capabilities.

Admittedly, these changes have introduced some instability over the past six months. Rest assured, these are growing pains on our path to maturity, and I am fully committed to steering Spikster back to a state of stability and reliability. 

I am confident in the long-term benefits it will yield. These include heightened security, increased efficiency, and a more robust architecture that will elevate Spikster to the pinnacle of hosting software solutions.

I want to extend an open invitation for dialogue, so please share any issues you've encountered, and I would be more than willing to walk them through with you. Additionally, you are always welcome to join our Discord server, where you can pose questions and directly reach out to me.

**Comment by donniewr** [comment]
> I want to extend an open invitation for dialogue, so please share any issues you've encountered, and I would be more than willing to walk them through with you. Additionally, you are always welcome to join our Discord server, where you can pose questions and directly reach out to me.

I never managed to test the panel because the passwords at the end of the installation are not correct. There is also a broken image on the forgot password page.

These details don't make me want to use the panel for my sites yet, but I hope you succeed in making it great.

**Comment by donniewr** [comment]
@yolanmees 

Do you think you can add PHP 8.2/8.3 to the official CIPI repo? I noticed that PHP 8.2 is installed on my CIPI server. It seems we just need to edit the interface to support it.

If you propose the change, they might accept it.
